api/citations/utils.py

```diff
@@ -30,8 +30,9 @@ def preprint_csl(preprint, node):
     csl = node.csl
 
     csl['id'] = preprint._id
-    csl['publisher'] = preprint.provider.name
+    csl['publisher'] = 'OSF Preprints' if preprint.provider.name == 'Open Science Framework' else preprint.provider.name
     csl['URL'] = display_absolute_url(preprint)
+    csl['keyword'] = 'preprint'
```

Comment:
`keyword` variable to tell `project` apart from `preprint` citations

Reply:
This may be unnecessary if my other comments are actionable